### ISSUE TITLE
Collectibles page shows loading collectibles always

### ISSUE THREAD

**Issue by noeliaSD** [status-im/status-desktop#13776]
### Description
The following account only has one collectibles, it's loaded correctly but the list of loading collectibles keeps visible forever:

https://github.com/status-im/status-desktop/assets/97019400/787c7e21-8302-494a-98cb-ce93fc3683bb

Reproduced in: **2.27.0-rc5**

**Comment by caybro** [comment]
Yeah, same here; I just got the 1 correct airdropped collectible, and everything else under "Community minted" is just showing this endless loading state

**Comment by alaibe** [comment]
@dlipicar i think this is a duplicate and has already been fixed?

**Comment by dlipicar** [comment]
@alaibe yeah this is most likely what I fixed in master. We might want to cherrypick it for the next 2.27.0 rc

**Comment by dlipicar** [comment]
@caybro hmm I've been testing collectibles for a while and haven't seen this. Do you have API keys set for all providers (Rarible, Opensea, Alchemy)? Is this on testnet or mainnet?

**Comment by caybro** [comment]
> @caybro hmm I've been testing collectibles for a while and haven't seen this since that fix. Do you have API keys set for all providers (Rarible, Opensea, Alchemy)? Is this on testnet or mainnet?

Yes and mainnet:
![image](https://github.com/status-im/status-desktop/assets/5377645/08cfea31-5023-4545-9ef8-56e3eb54eb87)

**Comment by dlipicar** [comment]
@caybro please paste the account address

**Comment by caybro** [comment]
> @caybro please paste the account address

0x1df966d5cd30a6b4cdb0067d9fba4cbf490b3d51 (mine) and 0x71207091db6f13ae3684bfd71104cf04c7c51670 (the watched account)

**Comment by dlipicar** [comment]
@caybro hmm couldn't reproduce on latest master...

It showed the loading animation for a while but then it disappeared
![image](https://github.com/status-im/status-desktop/assets/11161531/6d1ffbe2-995b-4124-8c3a-338069a44e2a)

Please enable debug logs, and share your app log and geth.log files
